server/src/utils/analytics.js

```diff
@@ -0,0 +1,18 @@
+const Analytics = require('analytics-node')
+const config = require('config')
+const { get } = require('lodash')
+const mongoose = require('mongoose')
+const UserWallet = mongoose.model('UserWallet')
+
+const trackDepositSuccess = async ({ walletAddress }) => {
+  const userWallet = await UserWallet.findOne({ walletAddress })
```

Comment:
can you wrap this in try/catch. so it won't break the deposit flow?

Reply:
Yeah sure, thanks for the feedback